Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -77,6 +77,7 @@
     <string name="enable_cheats">Enable Cheats</string>
     <string name="speed_limit">Speed Limit (0% = Unlimited)</string>
     <string name="overclock_warning">WARNING: Changing this from the default (100%) WILL break games and cause glitches. Please do not report bugs that occur with a non-default clock.</string>
+    <string name="achievements_submenu">Achievements</string>
```

Comment:
I would suggest changing this to "RetroAchivements". The fact that achievements are provided by RetroAchievements should be visible to the user at some point before they enter their username and password, so they know what they're logging in to.